config.properties

```diff
@@ -0,0 +1,4 @@
+#Thu Feb 16 13:11:56 CET 2017
+autosave_interval=120
+autosave_enabled=false
+autosave_path=D\:\\
```

Comment:
System.getProperty("user.home")   <- user home directory which is system independent